src/App.js

```diff
@@ -3,15 +3,18 @@ import TodoList from "./TodoList";
 import AddTodoForm from "./AddTodoForm";
 
 function App() {
-  const [newTodo, setNewTodo] = useState(""); // state for newTodo
+  const [todoList, setTodoList] = useState([]);
+
+  function addTodo(newTodo) {
+    setTodoList([...todoList, newTodo]);
+  }
 
   return (
-    <>
+    <div>
       <h1>ToDo List</h1>
-      <AddTodoForm onAddTodo={setNewTodo} /> {/* pass onAddTodo */}
-      <p>New task: {newTodo}</p> {/* display newTodo */}
-      <TodoList />
-    </>
+      <AddTodoForm onAddTodo={addTodo} />
```

Comment:
Nothing to change here, just my pet peeve regarding how they have you name stuff. I just think it's confusing to do it this way as `onAddTodo` is a bad function name because that's not what it does.

```suggestion
      <AddTodoForm addTodo={addTodo} />
```